### DISCUSSION TITLE
brew commands not working on big sur

### DISCUSSION THREAD

**Discussion by reyesvicente** [Homebrew/discussions#142] [Everyday usage] [answered]
Anyone encounter this and have a solution for this? I've downloaded the command line tools on apples website but still doesn't work

`$ brew upgrade                                                        
env: Snitch.app/Contents/Components:/Library/Apple/usr/bin:/Applications/Postgres.app/Contents/Versions/latest/bin: No such file or directory`

**Comment by reyesvicente** [comment, marked as answer]
Had to remove `alias brew="env PATH=${PATH//$(pyenv root)\/shims:/} brew"` from my `.zshrc` and it worked

**Comment by darthShadow** [comment]
The actual problem is the space in the path introduced by `Little Snitch` btw. You can simply remove the `Little Snitch.app` from the path and that should also solve the issue without removing the pyenv root (which you must have added for a specific reason).

**Reply by reyesvicente** [reply]
I forgot to add that I couldn't see where `Little Snitch` is placed in all my `.*rc` files. I don't know yet what's wrong why it's not showing.

**Reply by watchity** [reply]
Same. I don't know who injects Little Snitch into the path. My question is whether it is really needed in the PATH or not. Hmm...

**Comment by darthShadow** [comment]
I use this in my `.zshrc`:

```
function path_remove () {
  # Delete path by parts so we can never accidentally remove sub paths
  PATH=${PATH//":$1:"/":"} # delete any instances in the middle
  PATH=${PATH/#"$1:"/} # delete any instance at the beginning
  PATH=${PATH/%":$1"/} # delete any instance in the at the end
}
path_remove "/Applications/Little Snitch.app/Contents/Components"
```

This section should be added after the path is set.

**Comment by SPlatten** [comment]
What is the status on Homebrew and Big Sur 11.0.1 ?

I'm trying to get MySQL Workbench to communicate with MariaDB.  (mariadb installing using Homebrew).

**Reply by SMillerDev** [reply]
https://github.com/Homebrew/homebrew-core/issues/64785#issuecomment-727841171
